app/gameengine.py

```diff
@@ -3732,6 +3795,25 @@ def archive_revealed_cards_continuation():
                         "available_targets": available_targets,
                         "continuation": self.continue_resolving_effects,
                     })
+            case EffectType.EffectType_MultipleDieRoll:
+                effect_player.last_die_roll_results = [] # reset the results
+
+                amount = effect["amount"]
+                match amount:
+                    case "per_two_mascots":
+                        holomems = effect_player.get_holomem_on_stage()
+                        mascots = [attached for holomem in holomems for attached in holomem["attached_support"] if is_card_mascot(attached)]
+                        amount = len(mascots) // 2
+
+                die_effects = effect["die_effects"]
+                roll_effects = []
+                for _ in range(amount):
+                    roll_effects.extend(deepcopy(die_effects))
+
+                add_ids_to_effects(roll_effects, effect_player_id, effect["source_card_id"])
+                self.add_effects_to_front(roll_effects)
+                self.continue_resolving_effects()
```

Comment:
there's no reason to call continue_resolving_effects, this function was called by that anyway and you're already in the effect resolution loop.
So don't do this or set passed_on_continuation